src/Pages/SignUpPage/SignUpPage.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Navbar } from '../../Components/Navigation/Navbar'
+import { SignUp } from '../../Components/SignUp/SignUp'
+
+const SignUpPage = () => {
+  return (
+    <div>
+        <Navbar/>
```

Comment:
instead of reusing the component again and again you can use this component directly in app.js

Reply:
okey